metrics/factory.go

```diff
@@ -0,0 +1,62 @@
+package metrics
+
+import (
+	"net/url"
+	"time"
+
+	"github.com/socialpoint-labs/bsk/server"
+)
+
+// NewMetricsRunnerFromDSN creates a new metrics publisher and returns its Metrics
+// and Runner from a DSN configuration. If the configuration is not valid it panics.
+func NewMetricsRunnerFromDSN(DSN string) (Metrics, server.Runner) {
```

Comment:
For now I would leave it, but lets create an issue to not forget it: https://github.com/socialpoint-labs/bsk/issues/24